jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/python/PythonLambdaHandlerResolverTest.kt

```diff
@@ -140,6 +141,21 @@ class PythonLambdaHandlerResolverTest {
         assertHandler("app.handle", true)
     }
 
+    @Test
+    fun pyTestHandlersIgnored() {
+        createPyTestHandler("hello_world/app.py")
+
+        assertHandler("hello_world/app.handle", false)
+    }
+
+    @Test
+    fun handlesInDirectoriesMarkedTestNotFound() {
+        val vfs = createPyTestHandler("hello_world/app.py")
```

Comment:
This shouldnt use createPyTestHandler, it conflates the test with the test based check